docs/guide/apm-getting-started.asciidoc

```diff
@@ -97,13 +98,14 @@ output:
     hosts: ElasticsearchAddress:9200
 ----------------------------------
 
-
+[float]
 ==== Secure access to the API
 The HTTP API exposed by APM Server listens on `localhost` and port 8200 by default.
 If you change the listen address from `localhost` to something that is accessible from outside of the machine, we recommend setting up firewall rules to ensure that only your own systems can access the API.
-Alternatively, you can use the _secret token and TLS_ to secure access to APM Server API.
+Alternatively, you can use the https://www.elastic.co/guide/en/apm/server/current/_security.html[secret token and TLS] to secure access to APM Server API.
```

Comment:
We should use asciidoc attributes to resolve link paths that point to the docs. I've created an issue to track some cleanup work that's required. Doesn't have to happen right away...but before GA.